### DISCUSSION TITLE
FixedNav 组件

### DISCUSSION THREAD

**Discussion by oasis-cloud** [jdf2e/nutui-react#40] [General]
1. Event 名称加 on 前缀
2. 移除无用的 import
3. change 和 selected 类型建议写成形如 (value: boolean) => void 的类型，避免直接使用 Function 类型
4. type 属性增加 类型 type xx = ‘right' | 'left', postion  属性也需要修改，这样使用时代码提示更友好
5. 需要支持 calssName、style 属性，以及 rest 属性，参考 overlay ，搜索 rest 查看示例

**Comment by xiaoyatong** [comment]
1. interface 里的可选参数加上?: 进行标记；比如change就是必选，overlay可选；
2. navList 明确给出数据结构；如{id: '', num: '', icon: ''}，同样注意哪些是可选参数；还有当icon为空时，页面如何展示，没有托底icon。
